### ISSUE TITLE
Convert the project into an unique project

### ISSUE THREAD

**Issue by Mayank0255** [Mayank0255/Stack-Overflow-Clone#14]

**Comment by Mayank0255** [comment]
On hold right now

**Comment by adarshaacharya** [comment]
Hello @Mayank0255 !
I have created similar kind of app named **CsOverflow**. Have a look at it here.
https://github.com/adarshaacharya/CsOverflow

**Comment by Mayank0255** [comment]
@adarshaacharya 

Great job there man🎉

**Comment by adarshaacharya** [comment]
@Mayank0255 thanks buddy ! I was originally inspired from your repo and tried to implement on my own.

**Comment by Mayank0255** [comment]
@adarshaacharya Good to hear that man, keep up the good work :)

I can see you have used ORM on your project with PostgreSQL, I have an issue (#17) which is shifting from basic querying to an ORM Sequelize or Prisma with MySQL, so, would you like to work on it and help me out?

**Comment by adarshaacharya** [comment]
@Mayank0255 yes for sure ! I would look into the codebase first then decide how to migrate it to sequelize !

**Comment by Mayank0255** [comment]
> @Mayank0255 yes for sure ! I would look into the codebase first then decide how to migrate it to sequelize ! 

Great

If you have decided to work on it then just comment on the issue, so that I can assign it to you.

**Comment by Mayank0255** [comment]
@adarshaacharya 

Did you look into #17?

**Comment by adarshaacharya** [comment]
@Mayank0255 yes I've look it that. Actually I've exams coming up ! So, working on that issue would take more time. Instead I'm planning to work on #22 .

**Comment by Mayank0255** [comment]
@adarshaacharya Sure, no problem, just comment on the issue so that I can assign you that
